tools/cldr-code/src/license/THIRD-PARTY.properties

```diff
@@ -0,0 +1,6 @@
+# Generated by org.codehaus.mojo.license.AddThirdPartyMojo
+# Please fill the missing licenses for dependencies :
+# this may need to be updated before building final outputs
+# (for now)
+com.ibm.icu--icu4j-for-cldr--68.1-release-68-1=Unicode-DFS-2016
+com.ibm.icu--utilities-for-cldr--68.1-release-68-1=Unicode-DFS-2016
```

Comment:
ideally this would be updated when we bump the dependencies… the best solution is to improve things on the ICU side, ICU-21425